README.md

```diff
@@ -16,7 +16,10 @@ the Sia network.
 
 ## Prerequisites
 
-- [node & npm (packaged together)](https://nodejs.org/download/)
+- [siad](https://github.com/NebulousLabs/Sia)
+- [node & npm 6.9.0 LTS](https://nodejs.org/download/)
+Earlier node versions may work, but they do not have guarenteed support.
```

Comment:
Do our builds commit to using the LTS version of node? Want to make sure there will be a failure if something accidentally uses a new feature.